.github/actions/run-python-test-set/action.yml

```diff
@@ -144,10 +146,11 @@ runs:
         du -sh /tmp/test_output/*
 
     - name: Upload python test logs
-      if: always()
```

Comment:
` if: always()` IMO should stay, so that we upload logs for failed tests too.

Also, now GH Actions "Summary" tab gets those artifacts automatically and this is the only place to get the logs for debugging. With S3 this gets even more hardcode, so maybe we should keep the old action here for a while, before we set up some web UI for S3?

Reply:
Oh, that's great, thank you for sharing.
Not sure it's a good idea with S3, but good to know that we can send logs without compression and have easy to access links for their logs.